debian/changelog

```diff
@@ -1,3 +1,9 @@
+imunify-nsq (1.0.1-2) stable; urgency=low
+
+  * Added pidfile support
```

Comment:
Could you please describe here why we need support pid file option explicitly

Reply:
done